### PR TITLE
Crystallize example for the Data Fetching page

### PR DESCRIPTION
**Pull request by Plopix**
Hello Astro Team,

We thought it would make sense to add an example for [Crystallize](https://crystallize.com)

What do you think? thanks!

### CODE REVIEW

**Comment by hakonkrogh** [comment]
🙌

**Review by delucis — COMMENTED** [review]
Thanks for this @Plopix!

Because we don’t have room for full-blown guides for every service here, we’ve been linking to an external blog post or guide where available. Do you think there’s a chance we could add something like that for this too? Like a “How to get started with Astro and Crystallize” blog post?

**Comment by sarah11918** [comment]
Hi @Plopix! Thanks for this, just following up and I'm going to assign it to @Jutanium because we're actually going to start preparing larger CMS integration guides and you've already started the work for us! 

As @delucis mentioned, this page is really just to show small examples of fetching, so they can see what it looks like, but can't teach readers how to fully integrate their projects with an external CMS. So, we have been choosing examples where we can provide links to full tutorials that others have written. Do you have a blog post somewhere we could also include, to match the format of the other examples? If you don't have one, are you interested in creating one, maybe working with @Jutanium? 

He might not choose to include the example here on this page, but would be happy to help you create a full guide we can publish on its own page!

**Comment by Plopix** [comment]
hey @Jutanium, sorry I missed that comment. We had a blog post/page in mind already yes. But nothing done yet. but really wanting to do it. We'll work on a draft first, we can connect of course. Any particular thing you'd like to see ?

**Comment by sarah11918** [comment]
Hi @Plopix and @hakonkrogh! Sorry this has sat around for a bit, but I just wanted to let you know we're getting back to this one this week!

I'm going to ask @kevinzunigacuellar to be taking a look at this for conforming to fit with the other examples here, and to ask you any questions if he has more. As I mentioned earlier, part of the hold up has been that we are intending to include full guides for a handful of CMS partners, so we've been focusing on making a plan for that. And, that means this page might very well change entirely in the future. 

But, since we have a nice example already here, it would be a shame not to use it in the meantime! So I'll ask Kevin to jump in and do that!

File: src/pages/en/guides/data-fetching.md [hunk 1]
```diff
@@ -171,3 +171,42 @@ const data = await response.json();
...
+
+---
+<BaseLayout>
+  <h1>{response.tenant.name}</h1>
+	<nav>
+		<ul>
+      {navigation.tree.children.map(child => (
+        <li><a href={child.path}>{child.name}</a></li>
+      ))}
+		</ul>
```
Comment (kevinzunigacuellar) [inline src/pages/en/guides/data-fetching.md:209]: I was thinking that maybe we can keep the data fetching as short as possible to match other examples. I took the request example from https://crystallize.com/learn/developer-guides/catalogue-api/querying-the-catalogue

```suggestion
```astro title="src/pages/index.astro"
---
// Fetch your catalogue paths from Crystallize graphql API

import BaseLayout from '../../layouts/BaseLayout.astro';
import { createClient } from '@crystallize/js-api-client';

const apiClient = createClient({
    tenantIdentifier: 'furniture'
});

const query = `
  query getCataloguePaths{
    catalogue(language: "en", path: "/shop") {
      name
      children {
        name
        path
      }
    }
  }
`
const { data } = await apiClient.catalogueApi(query)
---
<BaseLayout>
  <h1>{catalogue.name}</h1>
	<nav>
		<ul>
      {catalogue.children.map(child => (
        <li><a href={child.path}>{child.name}</a></li>
      ))}
		</ul>
```

Comment (Plopix) [reply src/pages/en/guides/data-fetching.md]: it is fine with me. I just wanted to show that the client as some helpers versus simple GraphQL query. Bu that's totally fine.

File: src/pages/en/guides/data-fetching.md [hunk 2]
```diff
@@ -171,3 +171,41 @@ const data = await response.json();
 ```
 
 See the full tutorial [Building an Astro Website with WordPress as a Headless CMS](https://blog.openreplay.com/building-an-astro-website-with-wordpress-as-a-headless-cms) to add WordPress to your Astro Project! 
+
+### Example: Crystallize
+
+```astro title="src/pages/index.astro"
+---
+// Fetch your catalogue paths from Crystallize graphql API
```
Comment (sarah11918) [inline src/pages/en/guides/data-fetching.md:179]:
```suggestion
// Fetch your catalogue paths from Crystallize GraphQL API
```

**Comment by sarah11918** [comment]
Thanks everyone! I'll merge this into Docs now, and @Plopix, welcome to Docs as a first-time contributor! If you do get around to writing a full guide as a blog post, please do come back and PR the link here! We'll be working on adding CMS guides ourselves over the next few months, so we may also get in touch with you for a Crystallize one!

**Pull request merged**